akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -118,22 +118,24 @@ class IntroSpec extends TypedSpec {
     //#chatroom-gabbler
 
     //#chatroom-main
-    val main: Behavior[akka.NotUsed] =
+    val main: Behavior[String] =
       Actor.deferred { ctx ⇒
         val chatRoom = ctx.spawn(ChatRoom.behavior, "chatroom")
         val gabblerRef = ctx.spawn(gabbler, "gabbler")
         ctx.watch(gabblerRef)
-        chatRoom ! GetSession("ol’ Gabbler", gabblerRef)
 
-        Actor.immutable[akka.NotUsed] {
-          (_, _) ⇒ Actor.unhandled
+        Actor.immutablePartial[String] {
+          case (_, "go") ⇒
+            chatRoom ! GetSession("ol’ Gabbler", gabblerRef)
```

Comment:
why was this change needed?

Reply:
https://github.com/akka/akka/issues/24172